test-cases/scylla-operator/longevity-scylla-operator-3h.yaml

```diff
@@ -5,7 +5,9 @@ n_db_nodes: 3
 n_loaders: 2
 n_monitor_nodes: 1
 
-nemesis_class_name: 'KubernetesScyllaOperatorMonkey'
+nemesis_class_name: 'SisyphusMonkey'
+nemesis_include_filter: ['kubernetes']
+nemesis_seed: '026'
```

Comment:
Why nemesis seed is set?

Reply:
we set is to make sure we run the same sequence each time,
this is how SisyphusMonkey is used, so in less we add or remove a nemesis from the code, it would run 
the same sequence each time.

the idea is that if we run into something problematic, high chances that reruning we will hit the same issue (no always, but  that is the first thing we are being asked when investigating issue, and we want to be able to get to reproduction as fast as possible)

Again I'm just matching what we are doing for the rest of longevity for a few releases now (since 4.5 or 4.6)

If we are going to run some of those weekly, I would like to be aligned to how we do that in master.

Reply:
I know what is the effect of setting the seed. I asked "why?" because there is no reason for it now.
There is no sequence to repeat as of now.
Moreover, we will be able to discover more bugs running nemesis's in different order.

Reply:
We can leave one job which would be total random, but after a while it's stable enough (and not the otherway around)
I don't understand why you think there's no reason "now".
 
but generally I think we should be following the same logic we do for the release jobs.

Reply:
I got the answer. Thank you. Let it be as is.